gym/lib/gym/options.rb

```diff
@@ -249,6 +249,11 @@ def self.plain_options
                                      verify_block: proc do |value|
                                        UI.user_error!("Report output location not found at path '#{File.expand_path(value)}'") unless File.exist?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :analyze_build_time,
```

Comment:
Can you add at least one test for this new option to avoid regressions in the future?